articles/active-directory/roles/delegate-app-roles.md

```diff
@@ -80,6 +80,14 @@ Tips when creating and using custom roles for delegating application management:
 
 For more information on the basics of custom roles, see the [custom roles overview](custom-overview.md), as well as how to [create a custom role](custom-create.md) and how to [assign a role](custom-assign-powershell.md).
 
+
+
+## What to do when I do not have permission to register applications in the directory?
+
```

Comment:
```suggestion
If you can't register an application in the directory because your directory administrator has [restricted who can create applications](delegate-app-roles.md#restrict-who-can-create-applications), you should contact your administrator. The administrator can either [grant you permissions to create and consent to applications](delegate-app-roles.md#grant-individual-permissions-to-create-and-consent-to-applications-when-the-default-ability-is-disabled) by assigning you the Application Developer role, or create the application for you and then [assign you a built-in application administrator role](delegate-app-roles.md#assign-application-owners).
```